distribution/src/resources/runbit

```diff
@@ -15,11 +15,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-bin=`dirname "${BASH_SOURCE-$0}"`
-bin=`cd "$bin">/dev/null; pwd`
+# Invoke Drill using Java. Command line arguments are assumed to be
+# of the form
+#   -Dname=value
+# and are passed to Java as JVM arguments.
+#
+# Environment Variables
+#
+#   DRILLBIT_LOG_PATH       Path to the Drillbit log file.
+#                           (Set in drillbit.sh)
+#   DRILLBIT_QUERY_LOG_PATH Path the the query log file.
+#                           (Set in drillbit.sh)
+#   DRILL_JAVA_OPTS         Optional JVM arguments such as system
+#                           property overides used by both the
+#                           drillbit (server) and client,
+#                           set in drill-env.sh or the environment.
+#   DRILLBIT_JAVA_OPTS      Optional JVM arguments specifically for
+#                           the server (drillbit). Set in the
+#                           environment or in the user defined
+#                           drill-env.sh
+#   SERVER_GC_OPTS          Set in drill-env.sh, customized by
+#                           drillbit.sh.
+#   CP                      Drillbit classpath set in drillbit.sh
+
+#bin=`dirname "${BASH_SOURCE-$0}"`
+#bin=`cd "$bin">/dev/null; pwd`
+
+LOG_OPTS="-Dlog.path=$DRILLBIT_LOG_PATH -Dlog.query.path=$DRILLBIT_QUERY_LOG_PATH" 
+DRILL_ALL_JAVA_OPTS="$DRILL_JAVA_OPTS $DRILLBIT_JAVA_OPTS $SERVER_GC_OPTS $@ $LOG_OPTS"
```

Comment:
Glad to see `$@` in there. Finally there would be an option to override memory settings from the command line rather than through files. This will simplify Drill deployment on Mesos with Marathon.